<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAM (System Administrator Mate)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="1" column="1">
     <widget class="QLabel" name="error">
      <property name="styleSheet">
       <string notr="true">padding-left: 12px</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Компьютеры</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" colspan="6">
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Пользователи</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Администраторы</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Пользователи УРС</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>HDD</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Объём RAM</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CPU</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Время работы</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>ауе</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>localhost</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="5">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="6">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="7">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="8">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>123</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="5">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="6">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="7">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="8">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>456</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="5">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="6">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="7">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="8">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>123456</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="5">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="6">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="7">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="8">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>ауе123456</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="5">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="6">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="7">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="8">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Опрос машин каждые:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Опрос машин</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="spinBox">
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Показать данные о выбранном ПК</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Экспорт</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Управление выбранным ПК</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>минут</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Пользователи</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="4">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Экспорт</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Отображаемое имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID пользователя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Последняя смена пароля</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Активирован</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>ауе</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>123</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>456</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="2" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>123456</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="3" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>ауе123456</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="3">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="4" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Показать данные о пользователе</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Управление пользователем</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
